src/mots/directory.py

```diff
@@ -90,7 +89,7 @@ def load(self, full_paths: bool = False):
         people = list(self.config_handle.config["people"])
         self.people = People(people, {})
 
-    def query(self, *paths: str) -> QueryResult:
+    def query(self, *paths: str) -> "QueryResult":
```

Comment:
Just noticed that `paths` is actually `Tuple[str]` here, probably everywhere else we use `*args` as well.